webui/react/src/pages/TrialDetails/TrialInfoBox.test.tsx

```diff
@@ -10,14 +10,13 @@ import TrialInfoBox from './TrialInfoBox';
 
 vi.useFakeTimers();
 const setup = (trial: TrialDetails, experiment: ExperimentBase) => {
-  const view = render(
+  return render(
```

Comment:
no need to return `render`

```suggestion
  render(
```